conclave-host/src/main/kotlin/com/r3/conclave/host/internal/gramine/GramineEnclaveHandle.kt

```diff
@@ -71,6 +72,13 @@ class GramineEnclaveHandle(
         return process.inputStream.reader().use { it.readText() }.trimEnd()
     }
 
+    // ProcessBuilder.Redirect.DISCARD
```

Comment:
I don't understand the comment! Where is `Redirect.DISCARD` used?

Reply:
`Redirect.DISCARD` should be used instead of `NULL_FILE` but Conclave must support Java 8 and `Redirect.DISCARD` was only introduced in Java 9.

Reply:
Ok